README.org

```diff
@@ -366,6 +374,7 @@ All of the above settings can be applied on a per-project basis using [[https://
 | agent-shell-goose-command                | Command and parameters for the Goose client.                                  |
 | agent-shell-goose-environment            | Environment variables for the Goose client.                                   |
 | agent-shell-header-style                 | Style for agent shell buffer headers.                                         |
+| agent-shell-highlight-blocks             | Whether or not to highlight source blocks.                                    |
```

Comment:
@xenodium: Heads up that these unrelated entries resulted from me executing the `src` block earlier. Presumably these were left out of a previous PR? Either way, I'm happy to back these out if you think appropriate.